image/cli/mascli/functions/gitops_bootstrap

```diff
@@ -213,11 +213,11 @@ function gitops_bootstrap() {
   oc apply  -f ${DIR}/templates/gitops/bootstrap/namespace.yaml
 
   # 8. For some reason the application-controller needs restarting to pickup the RBAC changes
-  echo 'Restart application-controller'
-  oc rollout restart statefulset/openshift-gitops-application-controller -n openshift-gitops
+  # echo 'Restart application-controller'
+  # oc rollout restart statefulset/openshift-gitops-application-controller -n openshift-gitops
   
-  echo 'Waiting for rollout of application-controller to complete'
-  oc rollout status statefulset/openshift-gitops-application-controller -n openshift-gitops  
+  # echo 'Waiting for rollout of application-controller to complete'
+  # oc rollout status statefulset/openshift-gitops-application-controller -n openshift-gitops
```

Comment:
Remove these if we don't plan to do it anymore

Reply:
Removed the commented out lines.  We will monitor the fix (adding oc adm policy add-role-to-user admin)